### DISCUSSION TITLE
Can not upgrade Sonoff POWR320D

### DISCUSSION THREAD

**Discussion by jlazkano** [arendst/Tasmota#22038] [Q&A] [answered]
Hello,

I flashed a Sonoff POWR320D device, it has 12.5 firmware version, I try to upgrade from web interface but it doesn't change the version:

![image](https://github.com/user-attachments/assets/d2e1834c-f809-4958-a1c1-2626b3511289)

I am missing something?

Regards.

**Comment by sfromis** [comment, marked as answer]
Yeah, you missed the thread pinned at the top of the Discussions section, about upgrading from version 12.5. Short version is that you need to change the partition scheme, for allowing a larger binary.
